packages/spanner/src/transaction.js

```diff
@@ -371,7 +371,6 @@ Transaction.prototype.requestStream = function(config) {
       return;
     }
 
-    requestStream.end();
```

Comment:
@stephenplusplus so after a lot of digging, I believe we need to remove this line. This was originally a `destroy()` call but we decided to change it to `end()` because of #2358. However it turns out this is incorrect because gRPC uses the native stream module and there is no end method for the ClientReadableStream.

I looked into why destroy doesn't work, turns out there are a couple reasons:

1. gax gives back a Duplex stream which in turn will call `end()` when it is destroyed. As mentioned before there is no end method, so we get an error.
2. gRPC uses native streams, which means that users not running 8.0.0 won't have access to a destroy method since it was recently added.

I think our only course of action is to do nothing, which upon closer inspection seems correct.